app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,12 @@
+- provide(:title, "Log in")
+%h1 Log in
+.row
+  .col-md-6.col-md-offset-3
+    = form_for(:session, url: login_path) do |f|
+      = f.label :email
+      = f.email_field :email, class: 'form-control'
+      = f.label :password
+      = f.password_field :password, class: 'form-control'
+      = f.submit "Log in", class: "btn btn-primary"
+    %p
+      New user? #{link_to "Sign up now!", login_path}
```

Comment:
Did this come from somewhere? If so, :-1: to them for this string interpolation (rather than just a new line beginning with an `=`).